keylime/cloud_verifier_common.py

```diff
@@ -61,8 +61,7 @@ def init_mtls(section='cloud_verifier', generatedir='cv_ca'):
                 "To use tls_dir=generate, options ca_cert, my_cert, and private_key must all be set to 'default'")
 
         if generatedir[0] != '/':
-            generatedir = os.path.abspath(
-                '%s/%s' % (config.WORK_DIR, generatedir))
+            generatedir = os.path.join(config.WORK_DIR, generatedir)
```

Comment:
can we really get rid of `os.path.abspath()`? `os.path.abspath(os.path.join(config.WORK_DIR, generatedir))`